deno/deepgram_audio_summary/src/mod.ts

```diff
@@ -0,0 +1,35 @@
+export default async function (req: any, res: any) {
+    const APIkey = req.variables["DEEPGRAM_API_KEY"];
+    const fileUrl = req.payload["fileUrl"];
```

Comment:
to handle the payload as a string, make sure to call JSON.parse() on the payload:

```suggestion
    const { fileUrl } = JSON.parse(req.payload);
```

However, make sure to do error handling so that the function doesn't throw any exception.